python/pyspark/mllib/tests.py

```diff
@@ -818,6 +830,78 @@ def test_model_transform(self):
         self.assertEqual(model.transform([1.0, 2.0, 3.0]), DenseVector([1.0, 2.0, 3.0]))
 
 
+class StreamingKMeansTest(MLLibStreamingTestCase):
+    def test_model_params(self):
```

Comment:
@davies Could you please verify if these are the best way to add MLlib Streaming tests? I copied the idea from the tests in streaming.py.